docs/modules/operation/pages/deep-dive/flows/thresholding.adoc

```diff
@@ -2,14 +2,17 @@
 = Thresholding Flow Applications
 :description: Learn how to create thresholds on flows data in OpenNMS Horizon/Meridian to generate alarms when the traffic amount violates these thresholds.
 
-Flows are categorized in applications using the <<deep-dive/flows/classification-engine.adoc#ga-flow-support-classification-engine, Classification Engine>>.
+The <<deep-dive/flows/classification-engine.adoc#ga-flow-support-classification-engine, Classification Engine>> categorizes flows in applications.
 OpenNMS supports summing up the bytesIn/bytesOut data of flow records based on these flow applications.
 Users can define thresholds on this data to generate alarms when the amount of traffic violates these thresholds.
 
+[[configure-flow-thresholding]]
 == Configure thresholding for flow applications
 
-To enable thresholding for flows you need to enable the `applicationThresholding` flag in the adapter's definition.
-Note that thresholding on flow applications also works without enabling data collection on flow applications.
+To enable thresholding for flows you need to enable the `applicationThresholding` flag in the adapter's definition in `telemetry-configuration.xml`.
+
+Note that you do not need to enable or configure telemetryd on the core to support flows thresholding.
```

Comment:
This says you don't need to configure telemtryd on the core when using sentinels, but later on in the document it says sentinels receive threshold configs from the core.  This seems contradictory.
```suggestion
You do not need to enable or configure telemetryd on the core to support flows thresholding, if you are using Sentinels for processing telemetryd data.
```

Reply:
@fooker , which is true? Or what should we say here?

Reply:
For the wording, I'm with @mmahacek here. If you do not use sentine for this, you need to enable it of course.

Sentinel syncs the thresholds from core. But this is not done by telemetryd but the thresholding engine.